.gitignore

```diff
@@ -5,3 +5,4 @@
 .idea
 node_modules
 npm-debug.log
+bower_components
```

Comment:
Oops, I must have missed this message!

So, by JS library do you mean `script.js` and the associated JS files that we created?

> That seems like it would solve this problem (i.e. the 350map.org repo would include bower_components)

But then we still need a JS bundler to output one file that we can then include in the 350map.org site. And having a JS bundler will solve the original problem (we would no longer need to include the `bower_components` folder.

The way I see it, separating the JS would actually make development harder because we wouldn't be able to see our changes on 350map.org without first pushing changes to the library repo. And the library repo would still need an index.html so that we can see the map, which makes it pretty much just like the original repo.